### DISCUSSION TITLE
Dexcom one (byod) app / xdrip +

### DISCUSSION THREAD

**Discussion by Lovejoy74** [NightscoutFoundation/xDrip#2661] [General]
Every time I try and use the xdrip+ app alongside the byod app the byod app stopped receiving any information from my dexcom one, what am I doing wrong ? Any advice greatly appreciated,  thanks in advance

**Comment by Navid200** [comment]
The BYODA does not broadcast to xDrip for Dexcom One.  It does for G6.

If you must have BYODA for Dexcom One, until BYODA adds broadcast capability to xDrip, the only option is to use the Companion app mode if you can get BYODA to show notifications.

Do you know if it does?

**Reply by Navid200** [reply]
https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html

**Reply by Lovejoy74** [reply]
Tried this but xdrip still does not get any data, I do not think the byod app supplies the information in the information bar

**Reply by Navid200** [reply]
That's surprising.  Dexcom app creates notifications.  I don't understand why that feature would have been removed in BYODA.

**Reply by Lovejoy74** [reply]
It creates notifications but no data in them unless I have done something wrong

**Reply by Navid200** [reply]
If you have set up xDrip to collect directly from the transmitter, it will interfere with BYODA.  
You should set up xDrip to use the companion app mode.  have you done that?

**Reply by Lovejoy74** [reply]
Yes but then does not get any data

**Comment by Lovejoy74** [comment]
![Screenshot_20230205_214307_xDrip+](https://user-images.githubusercontent.com/124531903/216847740-89d2e66c-4af3-4d0d-a400-a6242906b1b5.jpg)
![Screenshot_20230205_214302_xDrip+](https://user-images.githubusercontent.com/124531903/216847744-28ca0bfd-b517-422b-92e7-6a6bdc09943f.jpg)
See above screenshot

**Reply by Navid200** [reply]
So, if you go to the hardware data source page and scroll all the way down and tap on disable collection, BYODA shows readings?  If you then enable Companion app mode, without changing anything else, BYODA stops showing readings?

**Comment by Lovejoy74** [comment]
My byoda app show the data bit only when I open it, xdrip receives no information at all no matter which setting I use

**Reply by Navid200** [reply]
Please!
xDrip is not going to show anything as long as BYODA is not showing notifications.  I would like to find out why it does not show notifications.  Can we stop mentioning that xDrip is not working?  I think that is clear now.

Can you go through all the settings inside BYODA to see if there is one that is needed to enable notifications?

**Reply by Navid200** [reply]
Do you absolutely need BYODA?
